redis/db/redisbp/example_hooks_test.go

```diff
@@ -23,7 +25,16 @@ func ExampleSpanHook() {
 		baseClient redis.Client
 	)
 	// Add the Hook onto baseClient
-	baseClient.AddHook(redisbp.SpanHook{ClientName: "redis"})
+	baseClient.AddHook(redisbp.SpanHook{
+		ClientName: "redis",
+		PromActive: &prometheusbp.HighWatermarkGauge{
+			HighWatermarkValue:   &prometheusbp.HighWatermarkValue{},
```

Comment:
nit: prefer `new()`, also making this a pointer was a mistake, if I made this a non-pointer then we won't need this line.